lib/components/user/account-setup-finish-pane.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const AccountSetupFinishPane = () => (
```

Comment:
This doesn't even need to be a function if the same thing is always returned.

Reply:
I think they should remain components. @landonreed?